rules/Arguments/ArgumentDefaultValueReplacer.php

```diff
@@ -113,15 +113,17 @@ private function processArgs(
             $replaceArgumentDefaultValue->getValueBefore()
         ) && $argValue === $replaceArgumentDefaultValue->getValueBefore()) {
             $expr->args[$position] = $this->normalizeValueToArgument($replaceArgumentDefaultValue->getValueAfter());
+            return $expr;
```

Comment:
Only return an Expr if the expr is actually changed, so that the rule is not applied if no change is made